librz/bin/bin.c

```diff
@@ -718,46 +718,31 @@ RZ_API RZ_OWN RzPVector *rz_bin_object_get_maps_at(RzBinObject *o, ut64 off, boo
 	return res;
 }
 
-RZ_API RzList *rz_bin_reset_strings(RzBin *bin) {
-	rz_return_val_if_fail(bin, NULL);
-	RzBinFile *bf = rz_bin_cur(bin);
-	if (!bf || !bf->o) {
-		return NULL;
-	}
-	return (RzList *)rz_bin_object_reset_strings(bin, bf, bf->o);
-}
-
-RZ_API RzList *rz_bin_get_strings(RzBin *bin) {
+RZ_DEPRECATE RZ_API RzList *rz_bin_get_strings(RzBin *bin) {
```

Comment:
Could you please add those `RZ_DEPRECATE` into the header file as well?

Reply:
they are already there.